packages/firestore-types/index.d.ts

```diff
@@ -57,6 +57,22 @@ export interface Settings {
   timestampsInSnapshots?: boolean;
 }
 
+/** Settings used to configure Firestore persistence. */
+export interface PersistenceSettings {
+  /**
+   * Whether to synchronize the in-memory state of multiple tabs. Setting this
+   * to 'true' in all open tabs enables shared access to local persistence,
+   * shared execution of queries and latency-compensated local document updates
+   * across all connected instances.
+   *
+   * To enable this mode, `synchronizeTabs:true` needs to be set globally in
+   * all active tabs.
```

Comment:
Seems like we should explain that /not/ setting this to true means enablePersistence() will fail in all but the first tab.

Reply:
Added this to the comment.